v63003/u_inref/gtm4212.csh

```diff
@@ -0,0 +1,39 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+#
+#
+
+$ydb_dist/mumps -run gtm4212
+
+set p231 = `cat temp231.out`
+set p230 = `cat temp230.out`
+set pge231 = `cat tempge231.out`
+set ple230 = `cat temple230.out`
+
+mkdir -p $p231
+mkdir -p $p230
+mkdir -p $pge231
+mkdir -p $ple230
+
+
+$gtm_tst/com/dbcreate.csh mumps 1 >>& dbcreate1.out
+echo "# Backing up Default to length 230 path (length of temp file is 24, so total path is 254)"
```

Comment:
Instead of checking for length 230, 231 and two other lengths, can we check for path lengths 229, 230, 231, 232 and 233 and have all of it in the reference file. That would simplify the csh script and the M program since we can use loops everywhere and would give us good test coverage every time this test runs. Thanks.